### PR TITLE
Add limit to max number of transactions

### PR DESCRIPTION
**Pull request by lipwei1808**

### CODE REVIEW

File: src/main/java/unicash/model/Model.java [hunk 1]
```diff
@@ -82,6 +82,11 @@ public interface Model {
      */
     void addTransaction(Transaction transaction);
 
+    /**
+     * Returns true if UniCash has reached max transaction capacity.
+     */
+    boolean isMax();
```
Comment (woojiahao) [inline src/main/java/unicash/model/Model.java:88]: Maybe a more descriptive name like `isFull`?

**Review by woojiahao — CHANGES_REQUESTED** [review]
Great turnaround, is the Amount restriction message change done?

File: src/main/java/unicash/model/Model.java (hunk 1 above)
Comment (lipwei1808) [reply src/main/java/unicash/model/Model.java:88]: Sure, let me update it

**Comment by lipwei1808** [comment]
> Great turnaround, is the Amount restriction message change done?

Yes, added it in already

**Review by woojiahao — APPROVED** [review]
LGTM, great work, now we don't have to worry about overflows

This PR addresses #143 formally

**Pull request merged**